src/main/java/it/gov/pagopa/payhub/auth/service/a2a/ClientServiceImpl.java

```diff
@@ -49,4 +50,11 @@ public Optional<Client> getClientByClientId(String clientId) {
 		return clientRetrieverService.getClientByClientId(clientId);
 	}
 
+	public ClientDTO verifyCredentials(String clientId, String clientSecret) {
```

Comment:
I found the word `authorize` more meaningfull in this context
```suggestion
	public ClientDTO authorizeCredentials(String clientId, String clientSecret) {
```